.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: java
+
+sudo: false
+
+# http://docs.travis-ci.com/user/caching/#Arbitrary-directories
+cache:
+  directories:
+  - $HOME/.m2
+
+jdk:
+  - oraclejdk8
+  - openjdk7
+
+matrix:
```

Comment:
As we are jointly reviewing https://github.com/ome/ome-common-java/pull/11, can you add a temporary step, cloning and installing this SNAPSHOT version to get Travis passing. Once we cut a new `ome-common` release, we can drop this step and consume the release version.